lua/expression3/extensions/math.lua

```diff
@@ -17,13 +17,32 @@ local extension = EXPR_LIB.RegisterExtenstion("math");
 -- TODO: Add math.pi as constant
 
 
+local PI = math.pi;
+local tan  = math.tan;
+local atan = math.atan;
 
 
+local function cot( radians )
+	return 1 / tan( radians );
+end
+
+local function acot( radians )
+	return PI/2 - atan( radians );
```

Comment:
Division is slow when it comes to math. So, either localize it or use `*0.5` instead.